example/styles/index.tss

```diff
@@ -0,0 +1,15 @@
+".container": {
+	backgroundColor:"white"
+}
+
+"Label": {
+	width: Ti.UI.SIZE,
+	height: Ti.UI.SIZE,
```

Comment:
Labels are SIZE by default

Reply:
@hansemannn this is just the default template of alloy

Reply:
Then we should remove it from there :)